### DISCUSSION TITLE
Is there a roadmap for new features for the Community Toolkit?

### DISCUSSION THREAD

**Discussion by mhershberg** [VsixCommunity/Community.VisualStudio.Toolkit#232] [General]
First off, this toolkit makes writing VSIX code much nicer than when I first tried my hand at it! Great job, everyone!

The current state of the toolkit is great for doing common tasks in VSIX code. Is there any kind of roadmap for planned new features? For example: any editor enhancements, custom language features (coloring, collapsible regions, Intellisense, code navigation and browsing), more services like TextSearchService, etc...

I'd love to contribute but I'm not sure what the goal for this project is or what parts of extension authoring are being looked at for potential coverage.

**Comment by madskristensen** [comment]
That's a great point, and no, there isn't a roadmap since a lot is determined by PRs sent by the community. However, I think it makes sense to open issues on areas not yet covered by the toolkit and then vote for them

**Reply by mhershberg** [reply]
Sounds good!

**Reply by mhershberg** [reply]
Is the appropriate place to open feature suggestion issues the Ideas section?

**Reply by madskristensen** [reply]
Yeah, it probably is and then we can discuss what exactly to do and then open an issue with the "specs"
